docs/en/ingest-management/fleet/migrate-elastic-agent.asciidoc

```diff
@@ -115,6 +115,38 @@ Using that value for `deployment-id`, the new {fleet-server} host URL is:
 [role="screenshot"]
 image::images/migrate-agent-fleet-server-host.png[Fleet server hosts showing the new host URL]
 
+[discrete]
+[[migrate-elastic-agent-reset-policy]]
+=== Reset the {ecloud} policy
+
+On your target cluster, certain settings from the original {ecloud} {agent} policiy may still be retained, and need to be updated to reference the new cluster. For example, in the APM policy installed to the {ecloud} {agent} policy, the original and outdated APM URL is preserved. This can be fixed by running the `reset_preconfigured_agent_policies` API request:
```

Comment:
```suggestion
On your target cluster, certain settings from the original {ecloud} {agent} policy may still be retained, and need to be updated to reference the new cluster. For example, in the APM policy installed to the {ecloud} {agent} policy, the original and outdated APM URL is preserved. This can be fixed by running the `reset_preconfigured_agent_policies` API request:
```